_upcoming-technologies/02-ipp-scan.md

```diff
@@ -1,6 +1,5 @@
 ---
 title: IPP scan (or virtual MF device) server (Scanner Application)
 ---
-The Printer Working Group (PWG) has also taken multi-function devices into account, devices which offer printer and scanner in one unit. For that they also have added IPP Scan, a driverless scanning standard for the scanners in IPP network multi-function devices.
-As IPP is already used by the operating systems for many years via CUPS, we also want to use it as scanning standard in the operating system, to easily integrate network multi-function devices and also for sandboxed packaging of scanner drivers. For this we emulate an IPP scanner, having a so-called Scanner Application, or even Printer/Scanner Applications being a driver for a whole multi-function device.
-Here we will discuss the transition from SANE-based scanning to IPP-based scanning in the operating system environment.
\ No newline at end of file
+IPP-Scan is an extension of the existing IPP protocol to facilitate driverless scanning. Currently, to scan in linux, a user has to procure the device specific SANE driver and then use a command line or a GUI based application that communicates with that scanner using this driver. Using IPP-Scan, a client would be able to communicate and receive files from an IPP-Scan enabled scanner using a universal IPP-Scan SANE backend or more directly through libcups, thus eliminating the need of maintaining scanner specific drivers. 
```

Comment:
`through libcups` -> `via an IPP implementation, like for example libcups`

Reply:
changed!